templates/api-deploy/deploy/ps/version.ps1

```diff
@@ -0,0 +1,15 @@
+#Docker variables
+$version = "1.0-local"
+$dotnetsdk = "6.0"
+$dotnetframework = "6.0"
+
+#Build variables
+$projectname = "EnerBank.Rebate" # needed for sln files
```

Comment:
not sure how much enerbank stuff you want/need to leave in

Reply:
updated